internal/build/sqlx_function_builder.go

```diff
@@ -134,15 +136,22 @@ func BuildGetFunction(
 		outputsWithError = append(outputsWithError, "nil")
 	}
 
+	queryType := false
+	if dialect == MySQL || dialect == SQLite3 {
+		queryType = true
```

Comment:
It's a very good idea. But `queryType` name isn't understandable. Please rename it to `specialQuery`.